addon/services/modal.js

```diff
@@ -21,7 +21,9 @@ export default class ModalService extends Service.extend(Evented) {
 
 		this.trigger('open', model);
 
-		return model.get('promise');
+		return new Promise((resolve) => {
```

Comment:
¿What happens if a modal fails when open? This promise never fulfills

Reply:
I have tried to try several points where it can fail, and I have not succeeded.

1. Modal open and rejects
2. Make a reject of the open in the constructor of the modal.


I can't think of any more cases. Do you know any more possible cases?